MANIFEST.in

```diff
@@ -1,3 +1,2 @@
 include README.md
 include LICENSE
-include requirements.txt
```

Comment:
Could you put this back? I know it is a duplication. But, different people has different workflows. Some uses setup.py, some uses requirements.txt, which can be useful information even for just looking at.

Plus, it can be used in conda to install the requirements too.